.github/ISSUE_TEMPLATE/01_crash.md

```diff
@@ -8,6 +8,9 @@ labels: bug: crash,
 
 
 
+<!-- If discussed in #zulip-terminal or another channel on chat.zulip.org, paste link below: -->
```

Comment:
I did consider this, but I didn't want to get too technical for issue filing - ie. a link is a good starting point if not familiar with Zulip.

For ZT in particular, we could even simplify it back to something like 'name of topic', since we don't have a 'copy link to message' (or topic, or stream) as yet.

I'm inclined to leave this as a followup for now.